tests/lib/prepare.sh

```diff
@@ -107,8 +108,9 @@ EOF
 #!/bin/sh -ex
 mount -t tmpfs none /tmp
 cp /bin/busybox /tmp
+cp $IMAGE_HOME/$IMAGE /tmp
 # blow away everything
-/tmp/busybox dd if=$IMAGE_HOME/$IMAGE of=/dev/sda bs=4M
+/tmp/busybox dd if=/tmp/$IMAGE of=/dev/sda bs=4M
```

Comment:
Is `$IMAGE_HOME` the original `/dev/sda`? If so, should we umount it before blasting it off?